.release/buildspec_e2e.yml

```diff
@@ -19,6 +19,15 @@ batch:
         variables:
           TEST_SUITE: addons
           APP_REGION: us-west-2
+    - identifier: s3
```

Comment:
Ooh what do you think of moving all this code into the existing `addons` test suite?  
(Launch both an Aurora DB and S3 bucket, then test that the API works)

In the past we ran into issues when adding too many e2e tests within the same region, VPC limits, polling limits... so we try to group related functionality under the same test suite.